File: ngDesk/ngDesk-Manager/src/main/java/com/ngdesk/Customer.java
package com.ngdesk;

import java.sql.Timestamp;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonProperty;

public class Customer {

	@JsonProperty("LAST_SEEN")
	@JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")
	private Timestamp lastSeen;

	@JsonProperty("STATUS")
	private String status;

	@JsonProperty("CUSTOMER_ID")
	private String customerId;

	@JsonProperty("CUSTOMER_UUID")
	private String customerUUID;

	@JsonProperty("EMAIL_ADDRESS")
	private String emailAddress;

	@JsonProperty("FIRST_NAME")
	private String firstName;

	@JsonProperty("DISABLED")
	private boolean disabled;

	public Timestamp getLastSeen() {
		return lastSeen;
	}

	public void setLastSeen(Timestamp lastSeen) {
		this.lastSeen = lastSeen;
	}

	public String getStatus() {
		return status;
	}

	public void setStatus(String status) {
		this.status = status;
	}

	public String getCustomerId() {
		return customerId;
	}

	public void setCustomerId(String customerId) {
		this.customerId = customerId;
	}

	public String getCustomerUUID() {
		return customerUUID;
	}

	public void setCustomerUUID(String customerUUID) {
		this.customerUUID = customerUUID;
	}

	public String getEmailAddress() {
		return emailAddress;
	}

	public void setEmailAddress(String emailAddress) {
		this.emailAddress = emailAddress;
	}

	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public boolean isDisabled() {
		return disabled;
	}

	public void setDisabled(boolean disabled) {
		this.disabled = disabled;
	}

}

File: ngDesk/ngDesk-UI/src/app/models/customer.ts
export class Customer {

  constructor(private EMAIL_ADDRESS: string, private PASSWORD: string, private FIRST_NAME: string,
    private LAST_NAME: string, private PHONE_NUMBER: string) {

  }

  public get emailAddress() {
    return this.EMAIL_ADDRESS;
  }

  public set emailAddress(emailAddress: string) {
    this.EMAIL_ADDRESS = emailAddress;
  }

  public get password() {
    return this.PASSWORD;
  }

  public set password(password: string) {
    this.PASSWORD = password;
  }

  public get firstName() {
    return this.FIRST_NAME;
  }

  public set firstName(firstName: string) {
    this.FIRST_NAME = firstName;
  }

  public get lastName() {
    return this.LAST_NAME;
  }

  public set lastName(lastName: string) {
    this.LAST_NAME = lastName;
  }

  public get phoneNumber() {
    return this.PHONE_NUMBER;
  }

  public set phoneNumber(phoneNumber: string) {
    this.PHONE_NUMBER = phoneNumber;
  }
}
